### ISSUE TITLE
Error in version 3.5.8?

### ISSUE THREAD

**Issue by glittle** [sboulema/TSVN#19]
When VS 2017 is opened (no solution), the "TSVN Pending Changes" window has this:

```
Exception details:
System.ArgumentException: The path is not of a legal form.
   at System.IO.Path.NormalizePath(String path, Boolean fullCheck, Int32 maxPathLength, Boolean expandShortPaths)
   at System.IO.Path.GetDirectoryName(String path)
   at SamirBoulema.TSVN.Helpers.CommandHelper.GetRepositoryRoot(String path)
   at SamirBoulema.TSVN.Helpers.CommandHelper.GetPendingChanges()
   at SamirBoulema.TSVN.TSVNToolWindow.OnToolWindowCreated()
   at Microsoft.VisualStudio.Shell.Package.CreateToolWindow(Type toolWindowType, Int32 id, ProvideToolWindowAttribute tool)
   at Microsoft.VisualStudio.Shell.Package.FindToolWindow(Type toolWindowType, Int32 id, Boolean create, ProvideToolWindowAttribute tool)
   at Microsoft.VisualStudio.Shell.Package.Microsoft.VisualStudio.Shell.Interop.IVsToolWindowFactory.CreateToolWindow(Guid& toolWindowType, UInt32 id)
   at Microsoft.VisualStudio.Platform.WindowManagement.WindowFrame.ConstructContent()

```

If the window is closed, then the `Tsvn/Windows/Pending Changes` menu shows a dialog:
![image](https://cloud.githubusercontent.com/assets/85186/24428180/ca8a1240-13ca-11e7-9e8b-0477df95840a.png)

**Comment by glittle** [comment]
This happened on two computers.
I've reverted to v3.4 and am fine for now.

**Comment by sboulema** [comment]
Sorry! Should be fixed in the new 3.6 release.
